doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -382,7 +382,9 @@ Binary libraries
 Bluetooth libraries and services
 --------------------------------
 
-|no_changes_yet_note|
+* :ref:`bt_fast_pair_readme` library:
+
+  * Updated the :c:func:`bt_fast_pair_info_cb_register` API to allow registration of multiple callbacks.
```

Comment:
Please provide Jira ticket ref in the commit message

Reply:
There is no JIRA ticket for this one

Reply:
I would link `NCSDK-30487` as it was done as a preparation step for this ticket.